<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QMainWindow" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>751</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>311</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>NMG</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>密  码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwardLineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="signBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sendBtn">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>用户名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userLineEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_pic_left">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>141</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pic_right">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>151</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 40pt &quot;Brush Script Std&quot;;</string>
      </property>
      <property name="text">
       <string>NMG</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_pic_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>131</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_detail">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>261</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_list">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>251</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="leval_fight">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>420</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>升级对战</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PVP_fight">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>480</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>决斗</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Britannic Bold'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>所有用户</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>在线用户</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>-10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 26pt &quot;Brush Script Std&quot;;</string>
      </property>
      <property name="text">
       <string>NMG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>540</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_list_user">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>251</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_win_rat">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>胜率：</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="table_list_ser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>251</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_detail_ser">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>281</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_pic_ser">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>131</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_begin_fight">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>480</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开始决斗</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 26pt &quot;Brush Script Std&quot;;</string>
      </property>
      <property name="text">
       <string>NMG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_ser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_re">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>480</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_pic_song">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>141</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_list_song">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 26pt &quot;Brush Script Std&quot;;</string>
      </property>
      <property name="text">
       <string>NMG</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_seg">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>451</width>
        <height>111</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;尊敬的用户：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;    非常遗憾！由于您输了决斗赛，请您选择一个精灵送出。但是请不要灰心，您可以通过升级赛不断磨砺您的精灵提升战力，欢迎再战！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirm">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>450</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>确认送出</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passwardLineEdit</tabstop>
  <tabstop>signBtn</tabstop>
  <tabstop>sendBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
